gulpfile.babel.js

```diff
@@ -17,6 +17,8 @@
  *
  */
 
+ // jscs:disable jsDoc
```

Comment:
Can we please get this just tossed directly into `master` ASAP? It is a major headace adding it in on all these branches to test things out repeatedly. The rest of this PR seems like extra semi-related fixes but not needed for the direct `gulp all` fails as far as I can tell.